project-resources/MEMBER_TRAVEL_FUND.md

```diff
@@ -14,6 +14,11 @@ and spread knowledge about and support the Javascript ecosystem and the Foundati
 * Candidates must be an active participant of one of the projects of the OpenJS Foundation.
 * Those requesting funds must indicate that they do not have funding available from another
   source, such as an employer or the event itself that might cover costs for presenters.
+* All projects in the foundation, regardless of stage or incubation status, are eligible to use the travel fund.
+* The fund can cover the costs, in whole or in part, of a contributor's hotel, airfare, and event registration (for OpenJS World and the Collab Summit)
+* Fund Applicants should seek to use employer funds as a first resort. Need is a factor.
+* Employees of Foundation Member companies are eligible, but we encourage the use of internal funds that may be available to them first.
```

Comment:
they seem to be pretty similar to:

> * Those requesting funds must indicate that they do not have funding available from another
  source, such as an employer or the event itself that might cover costs for presenters.	  source, such as an employer or the event itself that might cover costs for presenters.

is there a chance we can merge them together. Not sure if these 2 bullets add any new information.

Reply:
These all points are directly from [this issue which have been requested to be integrated into the document](https://github.com/openjs-foundation/summit/issues/249l. If you have further questions you can discuss them with @joesepi him and I have worked on this